pytorch_lightning/core/lightning.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, *args, **kwargs):
         self._datamodule = None
         self._results: Optional[Result] = None
         self._current_fx_name = ''
+        self._current_hook_fx_name = ''
```

Comment:
Can we make this `None` (ideally the one above as well, but this isn't from this PR)

Reply:
Great catch !